Lib/sqlite3/test/userfunctions.py

```diff
@@ -351,6 +298,24 @@ def test_empty_blob(self):
         cur = self.con.execute("select isblob(x'')")
         self.assertTrue(cur.fetchone()[0])
 
+    def test_func_params(self):
+        self.con.create_function("boomerang", 1, lambda x: x)
+        dataset = (
+            (42, int),
+            (1<<42, int),  # long long
```

Comment:
Please test also negative integers and an integer from 53 to 63 bits with non-zero lowest bits (it cannot be represented exactly as float).

Test also large integers outside of 64-bit integer range.